package.json

```diff
@@ -29,12 +30,14 @@
     "husky": "^8.0.2",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
+    "jest-sorted": "^1.0.14",
     "pg-format": "^1.0.4",
```

Comment:
ensure that `pg-format` is in `dependencies`, not `devDependencies`, lest your app break on deploy, when dev dependencies get pruned